charts/iap/values.yaml

```diff
@@ -15,9 +15,11 @@
 iap:
   image:
     repository: quay.io/oauth2-proxy/oauth2-proxy
-    tag: v6.0.0
+    tag: v6.1.1
```

Comment:
Please also adjust the appVersion in the Chart.yaml.

Reply:
updated version, thx for hint